test/org/sagebionetworks/bridge/models/studies/ConsentSignatureTest.java

```diff
@@ -207,7 +311,41 @@ public void jsonHappyCaseWithImage() throws Exception {
     }
     
     @Test
+    public void signatureContainsSignedOnValue() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(json, ConsentSignature.class);
+        assertTrue(sig.getSignedOn() > UNIX_TIMESTAMP);
+    }
+    
+    @Test
+    public void existingSignatureJsonDeserializesWithoutSignedOn() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(json, ConsentSignature.class);
+        assertEquals("test name", sig.getName());
+        assertEquals("1970-01-01", sig.getBirthdate());
+        assertTrue(sig.getSignedOn() > UNIX_TIMESTAMP);
+    }
+    
+    @Test
+    public void migrationConstructorUpdatesSignedOnValue() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(json, ConsentSignature.class);
+
+        ConsentSignature updated = new ConsentSignature.Builder().withConsentSignature(sig).withSignedOn(UNIX_TIMESTAMP).build();
+        assertEquals("test name", updated.getName());
+        assertEquals("1970-01-01", updated.getBirthdate());
+        assertEquals(UNIX_TIMESTAMP, updated.getSignedOn());
+        
+        try {
+            new ConsentSignature.Builder().withConsentSignature(sig).withSignedOn(DateTime.now().minusHours(4).getMillis()).build();
+        } catch(InvalidEntityException e) {
+            assertMessage(e, "signedOn", "signedOn must be a valid signature timestamp");
+        }
```

Comment:
Does this actually throw? The validator actually only checks for positive signedOn, so you'd need to pass in -1L or something in 1969 rather than 4 hours ago. Also, the builder replaces negative signedOn with now, so instead of throwing, this would still return a valid signature with an updated signedOn.

signedOn in the past should be allowed, for example, if we're deserializing an existing signature.